src/Services/Azure/Subscription/SubscriptionService.cs

```diff
@@ -12,6 +12,7 @@ public class SubscriptionService(ICacheService cacheService, ITenantService tena
     : BaseAzureService(tenantService), ISubscriptionService
 {
     private readonly ICacheService _cacheService = cacheService ?? throw new ArgumentNullException(nameof(cacheService));
+    private const string CACHE_GROUP = "subscription";
```

Comment:
The snake cased constants are a c# convetion break.

https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions

I think we need another PR to fix all of them

Reply:
https://github.com/dotnet/runtime/blob/fdbe38ce16671ae356e2bad21479e38b61d03340/src/libraries/System.Private.CoreLib/src/System/Char.cs#L48

```cs
        public const char MinValue = (char)0x00;

        private const byte IsWhiteSpaceFlag = 0x80;
        private const byte IsUpperCaseLetterFlag = 0x40;
        private const byte IsLowerCaseLetterFlag = 0x20;
        private const byte UnicodeCategoryMask = 0x1F;
```

Reply:
Opened https://github.com/Azure/azure-mcp/issues/180 to track the issue.